src/main/java/com/baloise/incubator/argonaut/application/github/GitHubWebhookRestController.java

```diff
@@ -114,7 +117,8 @@ private void handleIssueCommentEvent(GHEventPayload.IssueComment issueComment) {
         }
     }
 
-    private String getActionElementValue(JsonObject jsonObject) {
-        return jsonObject.get("action").getAsString();
+    private PullRequest createPullRequest(int id, GHRepository repository, GHPullRequest pullRequest) {
+        return new PullRequest(id, repository.getSvnUrl(), repository.getOwnerName(), repository.getName(), pullRequest.getHead().getRef(), pullRequest.getHead().getSha());
```

Comment:
There are a lot of parameters, maybe you can offer an additional constructor who has a repository as parameter, so you can reduce the parameters count.

Reply:
Another Question: What is the SvnUrl ? SourceVersioning ...?

Reply:
Thats how it is called in GitHub. 
Probably I need to come up with a property name thats fitting better. Its the clone URL, but without any git stuff. 
e.g. https://github.com/org/repo